### PR TITLE
auth_flow

### PR DESCRIPTION
**Pull request by kailash**

### CODE REVIEW

**Comment by kailash** [comment]
closes #90

File: src/test/resources/IUDX-File-Server-APIs-V3.5.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,3214 @@
...
+								"disabled": true
+							}
+						],
+						"body": {
+							"mode": "formdata",
+							"formdata": [
+								{
+									"key": "file",
+									"type": "file",
+									"src": "/home/kailash/Downloads/test.txt"
```
Comment (karun-singh) [inline src/test/resources/IUDX-File-Server-APIs-V3.5.postman_collection.json:457]: Need to revert back to the file path used in jenkins tests i.e- "/home/ubuntu/configs/test.txt"

Comment (karun-singh) [reply src/test/resources/IUDX-File-Server-APIs-V3.5.postman_collection.json]: @kailash need to do it for all apis that have the file path.

**Comment by karun-singh** [comment]
retest this please

**Comment by karun-singh** [comment]
retest this please

**Comment by karun-singh** [comment]
retest this please

**Pull request merged**